templates/index.html

```diff
@@ -25,111 +25,147 @@
+            <p class="p-matrix__desc">Apply user data to your instances automatically</p>
+          </div>
+        </li>
+      </ul>
+    </div>
+    <div class="row">
+      <h2>The Ubuntu OS</h2>
+    </div>
+    <div class="row">
+      <ul class="p-matrix">
+        <li class="p-matrix__item">
+          <a href="">
+            <img class="p-matrix__img" src="https://assets.ubuntu.com/v1/c5cb0f8e-picto-ubuntu.svg" alt="Icon">
+          </a>
+          <div class="p-matrix__content">
+            <h4 class="p-matrix__title"><a class="p-link" href="https://www.ubuntu.com/desktop">Desktop</a>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <a href="">
+            <img class="p-matrix__img" src="https://assets.ubuntu.com/v1/c5cb0f8e-picto-ubuntu.svg" alt="Icon">
+          </a>
+          <div class="p-matrix__content">
+            <h4 class="p-matrix__title"><a class="p-link" href="https://www.ubuntu.com/server">Server</a>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <a href="">
+            <img class="p-matrix__img" src="https://assets.ubuntu.com/v1/c5cb0f8e-picto-ubuntu.svg" alt="Icon">
+          </a>
+          <div class="p-matrix__content">
+            <h4 class="p-matrix__title"><a class="p-link" href="https://www.ubuntu.com/core">Core</a>
+          </div>
+        </li>
+      </ul>
     </div>
   </div>
-  <div class="p-strip is-shallow">
+
+  <div class="p-strip">
     <div class="row">
-      <h2>Externally hosted documentation</h2>
-      <table>
-        <thead>
-          <tr>
-            <th>Project</th>
-            <th>Description</th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr>
-            <td><a href="https://www.ubuntu.com/" class="p-link--external">Official Ubuntu documentation</a></td>
-            <td>
-              <p>Documentation for the Ubuntu operating system.</p>
-              <p><a href="https://help.ubuntu.com/" class="p-link--external">Read the official Ubuntu documentation</a></p>
-            </td>
-          </tr>
-          <tr>
-            <td><a href="https://jujucharms.com/" class="p-link--external">Juju</a></td>
-            <td>
-              <p>Documentation for Juju.</p>
-              <p><a href="https://jujucharms.com/docs/" class="p-link--external">Read the Juju documentation</a></p>
-            </td>
-          </tr>
-          <tr>
-            <td><a href="https://snapcraft.io" class="p-link--external">Snapcraft</a></td>
-            <td>
-              <p>Developer documentation to learn how to snap your apps for easy cross platform distribution and updates that just work.</p>
-              <p><a href="https://snapcraft.io/docs" class="p-link--external">Read the Snapcraft documentation</a></p>
-            </td>
-          </tr>
-        </tbody>
-      </table>
+      <h2>Creating documentation</h2>
+      <div class="row">
+        <div class="col-6">
+          <h3>Style guide</h3>
```

Comment:
I think we can remove the link below, and instead make this heading a link with an arrow, i.e:
`<h3><a href="etc">Title&nbsp;&rsaquo;</a></h3>`